src/main/java/org/takes/http/Options.java

```diff
@@ -57,7 +57,11 @@
     private final transient ConcurrentMap<String, String> map;
 
     /**
-     * Ctor.
+     * Constructs an {@code Options} with the specified arguments. The expected
```

Comment:
@essobedo Great description, but this particular case the constructor is package-protected, not a part of API
Let's leave it as is, but, usually, it's too much :)